pkg/manifests/types.go

```diff
@@ -186,6 +195,13 @@ type PrometheusK8sConfig struct {
 	Tolerations []v1.Toleration `json:"tolerations,omitempty"`
 	// Defines the pod's topology spread constraints.
 	TopologySpreadConstraints []v1.TopologySpreadConstraint `json:"topologySpreadConstraints,omitempty"`
+	// Defines the scraping profile which Prometheus will use to scrape metrics
+	// from the platform components. Supported values are `full` or `minimal`.
+	// In the `full` profile (default) Prometheus will scrape all the metrics
```

Comment:
strictly speaking, not all metrics are collected with the full profile as we've already made choices to drop some (e.g. kubelet, kube-state-metrics). Not sure how to phrase it differently though...

Reply:
I see, I changed it to `desired metrics` but I also have a hard time phrasing it differently without expending it too much